test/plugin_functional/plugins/core_app_status/public/plugin.tsx

```diff
@@ -26,6 +26,7 @@ import {
   CoreStart,
   AppMountParameters,
 } from 'kibana/public';
+import { renderApp } from './application';
```

Comment:
Can we add a comment that explains why this isn't an async import so that we don't accidentally make this flaky again?

Reply:
I removed a potential source of async logic here, but it's not the fix. It is: 
https://github.com/elastic/kibana/pull/72853/files#diff-2e5641ab42bbd0f3d15b213bde248322R111